src/transformState.js

```diff
@@ -1,11 +1,46 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- */
+// /**
+//  * @param {Object} state
+//  * @param {Object[]} actions
+//  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const dataToAdd = action.extraData;
+    const keysToRemove = action.keysToRemove;
+    const allKeys = Object.keys(state);
```

Comment:
```suggestion
```
redundant. looks like this variable needs only for clear type. but for remove all keys in state enough state
see comment below